src/Marketplace/Marketplace/help/Get-AzMarketplacePrivateStoreUserRule.md

```diff
@@ -84,7 +84,6 @@ Accept wildcard characters: False
 ```
 
 ### -ProgressAction
```

Comment:
```suggestion
### -ProgressAction
```
```suggestion
```

Reply:
Remove it from syntax as well.

Reply:
> Remove it from syntax as well.

It is no easy to remove "-ProgressAction" from cmdlet syntax